### PR TITLE
MTP-1554: upgrade checkboxwithsubsets

### PR DESCRIPTION
**Pull request by itemodahl**
MTP-1554

### CODE REVIEW

File: src/svelte/components/form/CheckboxWithSubSets.svelte [hunk 1]
```diff
@@ -21,6 +22,9 @@
     }
   }
 
+  $: fieldsetClass =
+    variation === 'primary' ? 'checkbox-subsets--primary' : 'checkbox-subsets--secondary'
+
   interface CheckboxWithSubSectionsStates {
     key: string
     docCount: number
```
Comment (lha047) [inline src/svelte/components/form/CheckboxWithSubSets.svelte:30]: I think we need to make this optional

File: src/svelte/components/form/CheckboxWithSubSets.svelte [hunk 2]
```diff
@@ -127,12 +130,12 @@
             on:change={() => mainCategory(mainIndex)} />
           <label for={listItem.key}>{`${listItem.displayName} (${listItem.docCount})`}</label>
```
Comment (lha047) [inline src/svelte/components/form/CheckboxWithSubSets.svelte:131]: Dont show the docCount when its not present. Same for the other docCount

**Review by lha047 — CHANGES_REQUESTED** [review]
Nice job :D

See comments

File: src/svelte/components/form/CheckboxWithSubSets.svelte (hunk 1 above)
Comment (itemodahl) [reply src/svelte/components/form/CheckboxWithSubSets.svelte:30]: What do you mean? It must be one of the two variations?

Comment (itemodahl) [reply src/svelte/components/form/CheckboxWithSubSets.svelte:30]: Ah you mean docCount...

**Pull request merged**